### PR TITLE
Secures application by:

### PR DESCRIPTION
**Pull request by hemantgoswami**
1) Only displaying errors that are alright to be viewed in prod
2) Redirects stack traces to log and brings user back to home page
3) Removes environment variable set that confuses with NODE_ENV which is the one node uses by default

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -64,13 +63,8 @@ app.use(passport.initialize())
...
 app.use(lusca.csrf())
-// app.use((req, res, next) => {
-//   if (req.path === '/api/upload') {
-//     next();
-//   } else {
-//     lusca.csrf(req, res, next);
-//   }
-// });
+app.use(lusca.xframe('SAMEORIGIN'))
+app.use(lusca.xssProtection(true))
```
Comment (Meepit) [inline app.js]: Why are these lines added again when they're already in the file?

Comment (hemantgoswami) [reply app.js]: removed duplicates
